person.rb

```diff
@@ -5,8 +5,8 @@ def initalize(_age, name = 'Unknown', parent_permission: true)
     @age = int
     @parent_permission = parent_permission
   end
-  attr_reader :id
   attr_accessor :name, :age
+  attr_reader :id
 
```

Comment:
- [ ] Please add the `GETTERS OR SETTERS` at the above of the `def initialize`, below the class Person.

![image](https://user-images.githubusercontent.com/55635977/157241022-ed137e58-4f8f-4e55-b09a-aa4c9500b6ea.png)

Reply:
@DelhinRharl As I stated above, please include the getters and setters at the above of the `def initialize` because getters and setters will be used or accessed later in the program. please don't put the getters or setters at the bottom of the class.